gnomad/resources/grch38/gnomad.py

```diff
@@ -483,9 +549,16 @@ def gnomad_gks(
         if custom_coverage_ht:
             coverage_ht = custom_coverage_ht
         else:
-            coverage_ht = hl.read_table(
-                coverage("genomes").versions[coverage_version].path
-            )
+            # in v4, only exomes have coverage
+            # in v3, only genomes have coverage (only genomes exist)
+            if high_level_version == "v3":
+                coverage_ht = hl.read_table(
+                    coverage("genomes").versions[coverage_version].path
+                )
+            else:
+                coverage_ht = hl.read_table(
+                    coverage("exomes").versions[coverage_version].path
```

Comment:
this is tricky, since v4 has genomes (from v3 but still) and exomes as well. could include the Data_Type parameter and say: 

if data_type==exomes: 
     if high_level_version == "v3": ERROR
     elif high_level_version == "v4": ....coverage("exomes")...
elif data_type==genomes:
     ...coverage("genomes")

but I'm not sure if the information in v4 genomes is different than v3 genomes ? I know it's the same samples but idk if it's identical

Reply:
just summing up coverage comments here:
I put in a fix to the coverage function while working on constraint so that it no longer tries to pull coverage versions that don't exist ( so you no longer have to assign "coverage_version" to avoid errors). v4 genomes is v3 genomes, so there are no new genome coverage files for v4. You can just pull call the coverage function and specify data_type and don't need to specify version. So for v4, using a datatype of either exomes or genomes will be fine since genomes will just pull the v3 coverage data. For v3, using a datatype of genomes is fine, but should error out if a user tries to pull exomes for v3.

"over_20" should also be added to the coverage annotation to be consistent with what's shown on the browser

Reply:
@theferrit32 it looks like coverage is the last item that still needs changes, can you edit the functions to address the above comment regarding coverage?

Reply:
@klaricch I'm about to push a change to handle that based on your comment above.

For the "over_20" comment, are you suggesting to add the `.over_20` field from the coverage_ht to the annotation output in addition to the `meanDepth` annotation?

@ahwagner @larrybabb do we want to add a field to include the `Fraction of individuals with >20x coverage` field added to the browser UI?

Reply:
yeah over_20 in additions to meanDepth

Reply:
@theferrit32 I'm fine with adding the two new fields, but I defer to @ahwagner for the final say. I'll make sure he's seeing these questions.

Reply:
@theferrit32 to make sure I'm following this proposal, is this an addition of `over_20` to the `caf.qualityMeasures` object? And no other additions? If so, this seems like the appropriate place to add the field and I have no issue with it so long as there is a PR to update the [model](https://github.com/ga4gh/va-spec/blob/1.0-alpha/schema/cohortAlleleFreq.yaml) and [examples](https://github.com/ga4gh/va-spec/blob/1.0-alpha/docs/Modeling/CohortFrequency/simple_result_example.yaml) at VA-Spec.

Reply:
@ahwagner https://github.com/ga4gh/va-spec/pull/124